ros2interface/ros2interface/verb/show.py

```diff
@@ -139,7 +186,25 @@ def add_arguments(self, parser, cli_name):
         arg.completer = type_completer
 
     def main(self, *, args):
+
+        if args.all_comments and args.no_comments:
+            return f"Cannot use the flags '{self._all_comments_flag}' and " \
+                   f"'{self._no_comments_flag}' at the same time"
```

Comment:
Instead of a custom check the two arguments should be in a mutual exclusive group (`add_mutually_exclusive_group`).